src/components/RequestedLessonsCard/components/ProposeLesson/index.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react'
+import {Text} from 'react-localize'
+import IconLabel from 'components/IconLabel'
+import ProposeLessonForm from './components/ProposeLessonForm'
+
+export default class extends Component {
+
+  state = {
+    isOpen: false,
+  }
+
+  handleClick = () => (
+    this.setState({
+      isOpen: true,
+    })
+  )
+
+  render() {
+    const {isOpen} = this.state
+    const {instructor} = this.props
+    return isOpen
+      ? <ProposeLessonForm instructor={instructor} />
+      : <div 
+          onClick={this.handleClick}
+          className='ttu'
+        >
+          <IconLabel
+            iconType='add'
+            labelText={<Text message='requestedLessons.proposeLesson.action' />}
+            color='blue'
+          />
+        </div>
+  }
```

Comment:
This `ProposeLesson` component now by default renders this:
![image](https://cloud.githubusercontent.com/assets/5497885/24430091/2fe51d36-13d2-11e7-889f-620f3c25a86f.png)

Then when tapped converts to the form:
![image](https://cloud.githubusercontent.com/assets/5497885/24430096/39538ea2-13d2-11e7-8e5f-e9140c901fd9.png)

Vojta plans to update the form to be a step-by-step process, but that will be done in the future in another PR and still needs discussion.